.github/workflows/close-stale-issues.yml

```diff
@@ -29,9 +31,9 @@ jobs:
 
         # These labels are required
         stale-issue-label: blocked/close-if-inactive
```

Comment:
Should we add [blocked/more-info-needed](https://github.com/aws/aws-sam-cli/labels/blocked%2Fmore-info-needed) label as well? Assuming we asked for more information which wasn't provided for certain time period.

Reply:
And also, [type/duplicate](https://github.com/aws/aws-sam-cli/labels/type%2Fduplicate) (which has 0 issues now but...)

Reply:
`stale-issue-label` gets added to the issues once it becomes "stale". I would prefer not to overload this too much and instead have a different label for stale-ness. So I will update this with a different label to signal the difference.